pkg/domain/infra/tunnel/images.go

```diff
@@ -19,7 +19,7 @@ import (
 	"github.com/containers/podman/v4/pkg/domain/entities/reports"
 	"github.com/containers/podman/v4/pkg/domain/utils"
 	"github.com/containers/podman/v4/pkg/errorhandling"
-	utils2 "github.com/containers/podman/v4/utils"
+	"github.com/docker/docker/pkg/archive"
```

Comment:
should this use `github.com/containers/storage/pkg/archive` instead?

Reply:
Yes. Thanks for catching. `vim-go` defaults to the `docker` one all the time.